### ISSUE TITLE
[linux] App doesn't receive clicks after moving window with startDragging method

### ISSUE THREAD

**Issue by ryota-hasegawa** [leanflutter/window_manager#57]
App doesn't receive any clicks after moving window with `windowManager.startDragging();`, and does not happens after dragging window title bar
I confirmed on latest stable and master branch of Flutter and I'm using `window_manager` `v0.1.4`
`bitsdojo_window` also has `startDragging` method, but this bug didn't happen with it.

<details>
<summary>Sample code</summary>

```dart
import 'package:flutter/material.dart';
import 'package:window_manager/window_manager.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Listener(
        onPointerDown: (_) => print('Clicked!'), // This won't be called if dragging has ended. It is the same with GestureDetector
        child: Scaffold(
          body: Center(
            child: GestureDetector(
              onTapDown: (_) => windowManager.startDragging(),
              child: Container(
                color: Colors.grey,
                padding: EdgeInsets.all(100),
                child: Text('Drag me'),
              ),
            ),
          ),
        ),
      ),
    );
  }
}
```
</details>

<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 2.8.1, on Zorin OS 16 5.13.0-25-generic, locale
    en_US.UTF-8)
    • Flutter version 2.8.1 at /home/user/Flutter/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (5 weeks ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[✓] Android toolchain - develop for Android devices (Android SDK version
    32.0.0-rc1)
    • Android SDK at /home/user/Android/Sdk
    • Platform android-32, build-tools 32.0.0-rc1
    • Java binary at: /home/user/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)
    • All Android licenses accepted.

[✗] Chrome - develop for the web (Cannot find Chrome executable at
    google-chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.22.1
    • ninja version 1.8.2
    • pkg-config version 0.29.1

[✓] Android Studio (version 2021.1)
    • Android Studio at /home/user/android-studio
    • Flutter plugin version 63.2.2
    • Dart plugin version 211.7798
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)

[✓] Connected device (1 available)
    • Linux (desktop) • linux • linux-x64 • Zorin OS 16 5.13.0-25-generic

! Doctor found issues in 1 category.
```
</details>

**Comment by lijy91** [comment]
Please see this example:

https://github.com/leanflutter/window_manager/blob/0ad29c5aab7fd4ec2af3007dd1bff54ec25cd85e/example/lib/pages/home.dart#L543-L565

We also have a `DragToMoveArea` Widget that you can also use

**Comment by damywise** [comment]
TL;DR
Don't use `onPointerDown`/`onTapDown `for `windowManager.startDragging()`
Use `onPanStart` instead

**Comment by ryota-hasegawa** [comment]
Thank you but I am having problem with startDragging method. So DragToMoveArea widget and onPanStart don't solve my issue

I updated sample code and the problem still happens
I think bitsdojo_window does something after dragging has ended but I have no idea

**Comment by lijy91** [comment]
Please don't wrap DragToMoveArea with GestureDetector

```dart
 GestureDetector( 
   behavior: HitTestBehavior.translucent, 
   onPanStart: (details) { 
     windowManager.startDragging(); 
   }, 
   onTap: () {
     // do something
   },
   onDoubleTap: () async { 
     bool isMaximized = await windowManager.isMaximized(); 
     if (!isMaximized) { 
       windowManager.maximize(); 
     } else { 
       windowManager.unmaximize(); 
     } 
   }, 
   child: Container( 
     margin: EdgeInsets.all(0), 
     width: double.infinity, 
     height: 54, 
     color: Colors.grey.withOpacity(0.3), 
     child: Center( 
       child: Text('DragToMoveArea'), 
     ), 
   ), 
 ), 
```

**Comment by ryota-hasegawa** [comment]
Thank you. I tried that but the problem still happens

I think this is a problem with startDragging, so the problem even happens with this code
```dart
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: DragToMoveArea(child: SizedBox()),
    );
  }
}
```

I should use this for sample all along. I'm sorry.
